microprofile/WFLY-17138-convert-mp-metrics-subsystem-to-legacy-subsystem.adoc

```diff
@@ -0,0 +1,104 @@
+= WFLY-17138 - Convert MP Metrics Subsystem to Admin-only Mode
+:author:            Jason Lee
+:email:             jasondlee@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+Red Hat has publicly stated our disagreement with the direction of MicroProfile Metrics and feel Micrometer, a de facto industry standard, provides a better metrics solution for our customers. To that end, Micrometer support will be added (via another JIRA) and, under this JIRA, MP Metrics support will be removed from standard configurations and module moved to "legacy" mode.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFLY-17138[WFLY-17138]
+
+=== Related Issues
+
```

Comment:
We should add the rleated product JIRA here, i.e. https://issues.redhat.com/browse/EAP7-1975

Reply:
Added